README.md

```diff
@@ -1,2 +1,45 @@
 # google_text
-Metacrate for Google Fonts text processing libraries
+
+Metacrate for Google Fonts text processing libraries.
+
+## overview
+
+This crate reexports compatible versions of Google Font's core text processing
+crates, which often have interdependencies and live in different repositories.
+By using this crate, you ensure that you are always using the latest set of
+compatible libraries.
+
+This is _only_ useful if you are using multiple high-level crates
+together, such as `write-fonts` and `skrifa`. If you only need to use one of
```

Comment:
harfrust and skrifa (the most likely problemmatic pairing) or harfrust, skrifa, klippa would be a better exmaple.